ts/Stock/Indicators/SMA/SMAIndicator.ts

```diff
@@ -481,7 +482,10 @@ class SMAIndicator extends LineSeries {
             indicator.redraw();
         }
 
-        indicator.isDirtyData = !!indicator.linkedSeries;
+        indicator.isDirtyData = !!indicator.linkedSeries.length;
```

Comment:
As a minor improvement to #17190, I added the `length` property, since `linkedSeries` could be an empty array.

Case: `!![] === true`